Makefile

```diff
@@ -19,6 +19,42 @@ SWAGGER_ENDPOINTS_SPEC = apps/aeutils/src/endpoints.erl
 
 PACKAGE_SPEC_WIN32 ?= ../ci/appveyor/package.cfg
 
+AE_DEB_VERSION=`cat VERSION`
+AE_DEB_DCH_REL_NOTE= \
+"Release notes are available in /usr/share/doc/aeternity-node/docs/release-notes/RELEASENOTES-`cat VERSION`.md"
+
+AE_DEB_EMAIL="info@aeternity.com"
+AE_DEB_NAME="Aeternity Team"
+
+
+# Packages from master MUST be pre-releases. Git master version
+# usually is always higher then the last stable release. However
```

Comment:
> Git master version usually is always higher then the last stable release.

I.e. usually when tagging&releasing v2.1.0 we shortly after (in a couple of days) bump VERSION in master as 2.2.0. Correct. Ok.